modules/core/src/main/scala/tethys/JsonReader.scala

```diff
@@ -11,15 +11,20 @@ trait JsonReader[@specialized(specializations) A] {
 
   def read(it: TokenIterator)(implicit fieldName: FieldName): A
 
+  def defaultValue: A
```

Comment:
I'm thinkingmaybe make it returning `Option[A]` and this would allow to privde default implementation and this wont' be breaking change

```suggestion
  def defaultValue: Option[A] = None
```

Reply:
Good idea.